### PR TITLE
Babble Module Candidate

### PR DESCRIPTION
**Pull request by dfgHiatus**

### CODE REVIEW

File: docs/hardware/babble.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Project Babble (Face Tracking)
```
Comment (benaclejames) [inline docs/hardware/babble.mdx:1]: Consider removing `(Face Tracking)`

File: docs/hardware/babble.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Summary
+
+[Project Babble](https://github.com/SummerSigh/ProjectBabble) is an open-source software project aiming to bring accessible face-tracking to PCVR headsets.
```
Comment (benaclejames) [inline docs/hardware/babble.mdx:5]: Consider adding a note/warning here that babble is only for mouth tracking and does not support eye tracking

File: docs/hardware/babble.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Set-up
+
+Setting up Project Babble is fairly simple. Please read carefully to avoid any issues. 
+
+:::note
+This guide assumes you have Project Babble installed, be it a release or the source code.
+It also assumes you have a compatible camera device. A standard webcamera will work, but you can also use a phone, etc.
```
Comment (benaclejames) [inline docs/hardware/babble.mdx:13]: `webcamera` -> `webcam`

**Review by benaclejames — CHANGES_REQUESTED** [review]

File: docs/hardware/babble.mdx (hunk 3 above)
Comment (dfgHiatus) [reply docs/hardware/babble.mdx]: Resolved in #[c2063f9](https://github.com/VRCFaceTracking/docs/pull/15/commits/c2063f9a32e0e1e23ea8fa10e6a546cc599908c9).

File: docs/hardware/babble.mdx (hunk 2 above)
Comment (dfgHiatus) [reply docs/hardware/babble.mdx]: Resolved in #[c2063f9](https://github.com/VRCFaceTracking/docs/pull/15/commits/c2063f9a32e0e1e23ea8fa10e6a546cc599908c9).

File: docs/hardware/babble.mdx (hunk 1 above)
Comment (dfgHiatus) [reply docs/hardware/babble.mdx]: Resolved in #[c2063f9](https://github.com/VRCFaceTracking/docs/pull/15/commits/c2063f9a32e0e1e23ea8fa10e6a546cc599908c9).

File: docs/hardware/Project Babble/babble-omnicept.mdx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+***But!*** With Project Babble this camera can be used to track your facial expressions! This guide will show you how to use the lower face camera with VRCFaceTracking.
+
+## Setup
+
+You will need the following software installed to use the lower face camera with VRCFaceTracking:
+- The [VRCFT-Babble Module](./babble-setup)
+- [OBS](https://obsproject.com/download)
+- [Project Babble](https://github.com/SummerSigh/ProjectBabble/releases/latest)
+- The [Glia Runtime/SDK](https://developers.hp.com/omnicept/downloads)
+- The [Glia Lip Viewer](https://discord.com/channels/849300336128032789/915075185328152606/1122604182130675772)
```
Comment (benaclejames) [inline docs/hardware/Project Babble/babble-omnicept.mdx:30]: Consider adding a warning that you must be a member of the discord to access this using `<CustomLink to="discord"/>`

Comment (dfgHiatus) [reply docs/hardware/Project Babble/babble-omnicept.mdx]: Replaced with an absolute link to `GliaLipViewer.zip` in [#7e8217c](https://github.com/VRCFaceTracking/docs/pull/15/commits/7e8217c2b17f53f736a68f24dedd74791c549a52)

**Review by benaclejames — APPROVED** [review]

**Pull request merged**